services/app-web/src/pages/SubmissionRevisionSummary/SubmissionRevisionSummary.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
🤔 💭 I'm wondering why make a separate component instead of reusing `SubmissionSummary`  tsx and referencing it at another route? Was that considered?

Reply:
To me `SubmissionSummary` seemed already a bit long and complex, adding more logic plus another banner (in another ticket) would add to that complexity. Although I have no issues if we wanted to reuse `SubmissionSummary`.